### PR TITLE
Regenerate the docs for pulumi v1.8.1

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: content/docs/reference/pkg/nodejs/pulumi/pulumi/asset/_index.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ git_sha: "03e0005fe0ce4b6bbe04c460a9bbc62869558214"
 
 <h2 id="apis">APIs</h2>
 <h3 class="pdoc-module-header" id="Archive" data-link-title="Archive">
-    <a href="https://github.com/pulumi/pulumi/blob/{{< param git_sha >}}/sdk/nodejs/asset/archive.ts#L21">
+    <a href="https://github.com/pulumi/pulumi/blob/{{< param git_sha >}}/sdk/nodejs/code/go/src/github.com/pulumi/pulumi/sdk/nodejs/asset/archive.ts#L21">
```
Comment (lukehoban) [inline content/docs/reference/pkg/nodejs/pulumi/pulumi/asset/_index.md]: These don’t look right - do these links work?

Comment (stack72) [reply content/docs/reference/pkg/nodejs/pulumi/pulumi/asset/_index.md]: Fixed - not sure why this happened before!

File: static/latest-version [hunk 2]
```diff
@@ -1 +1 @@
-1.7.1
\ No newline at end of file
+1.8.1
```
Comment (lukehoban) [inline static/latest-version]: Looks like there is still an extra newline here?

Comment (stack72) [reply static/latest-version]: Fixed!

**Pull request merged**